.github/workflows/R-CMD-check.yaml

```diff
@@ -62,19 +51,11 @@ jobs:
           http-user-agent: ${{ matrix.config.http-user-agent }}
           use-public-rspm: true
 
-      - name: Install system dependencies-2
-        if: runner.os == 'Linux'
-        run: |
-          sudo apt-get install -y libarchive-dev
-
-      - name: Install system dependencies-3
-        if: runner.os == 'macOS'
-        run: |
-          brew install libarchive
-
```

Comment:
Can you also do some research here to see why / when this was introduced? My hunch is that it was in a period when archive needed to be built from source in GHA. But I don't think this is a constant ongoing need.

Reply:
That seems right. Here is where it was first added

https://github.com/r-lib/vroom/commit/81afde37c968904c8e5c0caa633fedc7a00920f6

Reply:
Yeah, I believe we no longer need this. The deletion is good to go.